### DISCUSSION TITLE
Page Bundles not picking up md's in a subsection other than the index.md

### DISCUSSION THREAD

**Discussion by cwooley-rh** [luizdepra/hugo-coder#705] [Q&A]
Im sure this is just a lack of experience with the platform but i have a structure like such;

content/docs/
product1/feature1/index.md
...
product1/featureN/index.md
...
productN/featureN/index.md

where each product has a list page for its page bundle, however in some product sections there are more than one doc that is associated with a feature, I would like the product's list page to pick those up as well as they are important to add to the tag pages.

Do I need to create a layouts/<section>/list.html template for each product? Seems overkill

I appreciate the help and love the theme

**Comment by luizdepra** [comment]
You shouldn't have to do that. But I might be wrong because I never used this feature from Hugo.
Do you have an example of `content` or a public repository so I could test it?
